### DISCUSSION TITLE
Export/sync to Obsidian

### DISCUSSION THREAD

**Discussion by Spinkman** [karakeep-app/karakeep#622] [Ideas]
I'm loving the auto tagging, the chrome extension, and the share to hoarder from mobile. I am finding myself carrying duplicates in hoarder and in obsidian. Would be cool if there was way to link them together.

**Comment by MohamedBassem** [comment]
I 100% understand the need for something like this but unfortunately I don't have the capacity to implement it. With the release of the REST APIs in the next release, I hope someone in the community would be able to build the obsidian plugin and maintain it. I'd be more than happy to close the gap on any of the APIs needed to support this sync.

**Comment by jhofker** [comment]
I wanted this too, and started in on it. Pretty functional right now in my testing and it should be in the community plugins soon! It doesn't fix the duplicate situation, and in fact makes it "worse" depending on your point of view. However, it allows your hoarder bookmarks to show up in Obisidian, and for notes that you add in Obsidian on the bookmarks to sync back to Hoarder.

https://github.com/jhofker/obsidian-hoarder/

**Reply by MohamedBassem** [reply]
@jhofker That's great news! I know a lot of people are waiting for this. Once it's on the obsidian store, I'd be more than happy to feature in the community projects page!

**Reply by jhofker** [reply]
This is officially available! https://obsidian.md/plugins?id=hoarder-sync
